app/model/WaterData.js

```diff
@@ -0,0 +1,69 @@
+Ext.define('MoMo.client.model.WaterData', {
+
+    extend: 'Ext.data.Model',
+
+    proxy: {
+        type: 'rest',
+        url: '/momo/geoserver.action',
+        headers: BasiGX.util.CSRF.getHeader(),
+        extraParams: {
+            service: 'WFS',
+            version: '1.1.0',
+            request: 'GetFeature',
+            outputFormat: 'application/json',
+            typeName: 'momo:ecotech_data'
+        },
+        reader: {
+            type: 'json',
+            rootProperty: 'features',
+            transform: {
+                fn: function(data) {
+                    var result = [];
+                    Ext.each(data.features, function(feat) {
+                        result.push(feat.properties);
+                    });
+                    return result;
+                }
+            }
+        }
+    },
+
+    fields: [{
+        name: 'no',
+        type: 'int'
+    }, {
+        name: 'time',
+        type: 'date',
+        dateFormat: 'Y-m-d H:i',
+        convert: function(value){
```

Comment:
You may use http://docs.sencha.com/extjs/6.0.0/modern/Ext.data.field.Date.html#cfg-dateReadFormat instead of an convert function.